csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/OrtFloat16Tests.cs

```diff
@@ -0,0 +1,507 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Xunit;
+
+namespace Microsoft.ML.OnnxRuntime.Tests
+{
+    [Collection("Ort Float16 tests")]
+    public class OrtFloat16Tests
+    {
+        [Fact(DisplayName = "ConvertShortToFloat16")]
+        public void ConvertShortToFloat16()
+        {
+            ushort[] shortValues = { 0, 1, 2, 3, 4, 5, 6, 7 };
```

Comment:
I wonder if it is enough values.

Reply:
These values are in fact negative :), I checked the code coverage and the rounding code is covered too. I will add some code to check for rounding.